### ISSUE TITLE
za.co.absa.cobrix.cobol.parser.exceptions.SyntaxErrorException: Syntax error in the copybook at line 95: Invalid input '999.9999' at position 95:63

### ISSUE THREAD

**Issue by BharathBejoy** [AbsaOSS/cobrix#727]
## Background
I'm using za.co.absa.cobrix:spark-cobol_2.12:2.7.9 Maven Library in Azure Databricks to load Cobol (EBCDIC) files into Databricks. It works very well for all files except one. 

## Error Message
Py4JJavaError: An error occurred while calling o462.load.
: za.co.absa.cobrix.cobol.parser.exceptions.SyntaxErrorException: Syntax error in the copybook at line 95: Invalid input '999.9999' at position 95:63

## CPY File (Copybook)
![image](https://github.com/user-attachments/assets/ca459007-42f1-41d5-b984-536c46b46b55)

## Question
I'd appreciate some assistance in resolving this issue.

**Comment by yruslan** [comment]
Hi, thanks for the issue report! I'll try to reproduce it. 

However, 'VALUE' clauses are ignored by Cobrix. It never validates that the corresponding column has one of the listed values. So you can safely comment or delete line 95 to make it work.

For the parser check, could I ask you to send me lines 92-96 of the copybook? I see '1942' there and can't put it into the context.

**Comment by BharathBejoy** [comment]
> Hi, thanks for the issue report! I'll try to reproduce it.
> 
> However, 'VALUE' clauses are ignored by Cobrix. It never validates that the corresponding column has one of the listed values. So you can safely comment or delete line 95 to make it work.
> 
> For the parser check, could I ask you to send me lines 92-96 of the copybook? I see '1942' there and can't put it into the context.

Hi @yruslan, Please have a look at the attached image for the lines you asked for from the copybook.

![image](https://github.com/user-attachments/assets/5ed847b2-4f94-4411-81f8-ee0d7c8ddc97)

**Comment by yruslan** [comment]
The grammar has support for dots in numeric literals:
https://github.com/AbsaOSS/cobrix/blob/ae3fedff67eb9844abdc18383cfa58085d88f641/cobol-parser/src/main/scala/za/co/absa/cobrix/cobol/parser/antlr/copybookLexer.g4#L229-L229

I can reproduce the issue, but it is not obvious how to fix it. So will keep it open for now.

**Comment by yruslan** [comment]
Not at all. Changing that literal won't affect data in any way. Cobrix ignores `VALUE` clause.

It is an interesting issue. If the literal contains any digit but `9`, it passes. So you can replace `+999.9999` with `+999.99990` safely, or to any other number.

**Comment by yruslan** [comment]
Hi @tr11 , I hope you are doing well! This issue might be interesting to you too

**Comment by yruslan** [comment]
Hi @BharathBejoy , this issue is finally fixed in `2.10.8`. Sorry it took so long. At the and AI helped understand what was the issue with the parser and how can we extend its parsing of `88 SOMETHING VALUE +999.9999.`
